blackbox/docs/configuration.txt

```diff
@@ -752,15 +769,29 @@ Collecting Stats
 
     Using a very small value can cause a high load on the cluster.
 
-.. _stats.breaker.logs.limit:
+.. _stats.breaker.jobs_logs.limit:
 
-**stats.breaker.logs.limit**
+**stats.breaker.jobs_log.limit**
   | *Default:*    ``5%``
   | *Runtime:*   ``yes``
 
-  The logs circuit breaker allows CrateDB to prevent job logs data structures
-  from exceeding a certain amount of memory by removing old job logs entries.
+  The jobs log circuit breaker prevents that :ref:`sys.jobs_log <sys-logs>`
+  exceeds a certain amount of memory by removing old log entries. The value
+  specifies the maximum of memory that can be used out of :ref:`crate-heap-size`.
+  Reaching the limit logs an error message and causes :ref:`sys.jobs_log <sys-logs>` to be
+  cleared.
```

Comment:
just for my own understanding, by "clears" do you mean completely? or just enough to keep within the memory limits?